UML editor/src/main/java/codecain/GraphicalUserInterface/Controller.java

```diff
@@ -976,110 +255,35 @@ public void populateGUIFromClassMap() {
         System.out.println("GUI populated from class map.");
     }
 
-
- 
     /**
-     * Calculates and assigns a non-overlapping position for a `ClassNode` within the specified container.
+     * Selects a specific class node in the GUI, updating the selection state.
      *
-     * This method calculates an appropriate (x, y) position for the given `ClassNode`
-     * within the `container`. It ensures that no `ClassNode` overlaps with existing nodes
-     * by iterating over all current nodes and adjusting the position accordingly.
```

Comment:
Looks like you deleted the overlapping classes method. It looks like its called later, and is not available now